README.md

```diff
@@ -2,14 +2,24 @@
 
 # Personal Portfolio project
 
+> This Project is a clone of a Portfolio template from figma
+
 ## Built With
 
 - HTML/CSS
 - NPM, Git, Github
 
+## Live Demo
+
+[Live Demo Link](https://mromachi.github.io/Portfolio-Microverse/)
+
```

Comment:
-[Optional] Please also add a way to clone your repo. like git clone `<The repo link>` and then cd into the file folder of the cloned repo on your machine.  PLease it is to make your project more professional.